docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -0,0 +1,152 @@
+# Machine Learning Training
+
+<!-- Overview -->
+
+Machine Learning Training allows to construct, control and train a machine learning model in Tizen devices.
+Please note that, this feature remains experimental for Tizen 6.0 and is not full-fledged for every needs.
+
+## Prerequisites
+
+To enable your application to use the machine learning training functionality:
+
+1. To use the functions and data types of the Machine Learning Inference API, include the `<nntrainer.h>` header file in your application:
+
+   ```c
+   #include <nntrainer.h>
+   ```
+
+2. To use the Machine Learning Training API, include the following features in your `tizen-manifest.xml` file:
+
+   ```xml
+   <feature name="http://tizen.org/feature/machine_learning">true</feature>
+   <feature name="http://tizen.org/feature/machine_learning.training">true</feature>
+   ```
+
```

Comment:
add a statement about priveleges (if saving file outside the app, then apply privilege)?

Reply:
Done